### PR TITLE
Support dot in the tag names

### PR DESCRIPTION
**Pull request by scf2k**

### CODE REVIEW

File: scan/scanner.go [hunk 1]
```diff
@@ -37,7 +37,7 @@ import (
 const (
 	rxMethod = "(\\p{L}+)"
 	rxPath   = "((?:/[\\p{L}\\p{N}\\p{Pd}\\p{Pc}{}\\-\\._~%!$&'()*+,;=:@/]*)+/?)"
-	rxOpTags = "(\\p{L}[\\p{L}\\p{N}\\p{Pd}\\p{Pc}\\p{Zs}]+)"
+	rxOpTags = "(\\p{L}[\\p{L}\\p{N}\\p{Pd}.\\p{Pc}\\p{Zs}]+)"
```
Comment (fredbi) [inline scan/scanner.go]: Shouldn't the dot be quoted in regexp?

Comment (scf2k) [reply scan/scanner.go]: It does work that way too

Comment (gregmarr) [reply scan/scanner.go]: It works, but accepts any character, not just dot.

Comment (scf2k) [reply scan/scanner.go]: oops sorry, just fixed it

**Comment by casualjim** [comment]
Can you sign your pull request? 
https://help.github.com/articles/about-commit-signature-verification/

**Comment by scf2k** [comment]
> Can you sign your pull request?

done

**Pull request merged**